src/components/task-card.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { Card } from 'react-native-paper';
+import { MaterialCommunityIcons } from '@expo/vector-icons'
+import { paperTheme } from '../theme/paper-theme';
+import { Button } from './button';
+
+const TaskCardStyle = StyleSheet.create({
+    container: {
+        width: '100%',
+        elevation: 3,
+        marginVertical: 7,
+        flex: 1,
+        display: 'flex',
+        flexDirection: 'row',
+    },
+    iconContainer: {
+        width: 75,
+        justifyContent: 'center',
+        alignItems: 'center'
+    },
+    titleContainer: {
+        flex: 1,
+        alignItems: 'flex-start',
+        justifyContent: 'center',
+    },
+    distanceContainer: {
+        flex: 1,
+        flexDirection: 'row',
+        alignItems: 'center',
+        alignContent: 'center',
+        justifyContent: 'flex-start',
+    },
+    confirmButton: {
+        width: 50,
+        alignItems: 'center',
+        justifyContent: 'center',
+        backgroundColor: paperTheme.colors.primary
+    }
+});
+
+interface TaskCardProps {
+    owner: string,
+    tag: string,
+    iconName: string
```

Comment:
Maybe this should have a type with set options?

Reply:
Set options? 😮